framework/source/class/qx/data/controller/ListOfObjects.js

```diff
@@ -0,0 +1,425 @@
+/* ************************************************************************
+
+   qooxdoo - the new era of web development
+
+   http://qooxdoo.org
+
+   Copyright:
+     2004-2009 1&1 Internet AG, Germany, http://www.1und1.de
+
+   License:
+     LGPL: http://www.gnu.org/licenses/lgpl.html
+     EPL: http://www.eclipse.org/org/documents/epl-v10.php
+     See the LICENSE file in the project's top-level directory for details.
+
+   Authors:
+     * Martin Wittemann (martinwittemann)
+     * John Spackman (john.spackman@zenesis.com)
+
+************************************************************************ */
+
+
+/**
+ * <h2>List Controller</h2>
+ *
+ * *General idea*
+ * The list controller is responsible for synchronizing every list like widget
+ * with a data array. It does not matter if the array contains atomic values
+ * like strings of complete objects where one property holds the value for
+ * the label and another property holds the icon url. You can even use converters
+ * that make the label show a text corresponding to the icon, by binding both
+ * label and icon to the same model property and converting one of them.
+ *
+ * *Features*
+ *
+ * * Synchronize the model and the target
+ * * Label and icon are bindable
+ * * Takes care of the selection
+ * * Passes on the options used by {@link qx.data.SingleValueBinding#bind}
+ *
+ * *Usage*
+ *
+ * As model, only {@link qx.data.Array}s do work. The currently supported
+ * targets are
+ *
+ * * {@link qx.ui.form.SelectBox}
+ * * {@link qx.ui.form.List}
+ * * {@link qx.ui.form.ComboBox}
+ *
+ * All the properties like model, target or any property path is bindable.
+ * Especially the model is nice to bind to another selection for example.
+ * The controller itself can only work if it has a model and a target set. The
+ * rest of the properties may be empty.
+ *
+ * *Update from List*
+ *
+ * This controller is a replacement for {@link qx.data.controller.List} which is now
+ * deprecated; it is a drop in replacement for List with the exception of the delegate
+ * which no longer provides the filter or configureItem methods.  Filtering is
+ * achieved by using {@link qx.data.controller.ArrayFilter}. 
```

Comment:
fixed - the delegate is now IArrayDelegate, and configureItem is replaced with [un]bindTargetItem